generators/app/templates/src/app/_app.component.spec.ts

```diff
@@ -59,5 +65,5 @@ describe('AppComponent', () => {
     const fixture = TestBed.createComponent(AppComponent);
     const app = fixture.debugElement.componentInstance;
     expect(app).toBeTruthy();
-  }));
+  }), 30000);
```

Comment:
Why? Is there some timeout issues? 🤔

Reply:
I experienced some timeout issues caused by the Jest transformation part during some test cases. Let me check if it still needed